app/src/main/java/com/pajato/android/gamechat/exp/Checkerboard.java

```diff
@@ -45,14 +45,14 @@
 
     // Private class constants.
 
-    /** The amount of space taken by a smart phone toolbar in dp units. */
-    private static final int SMART_PHONE_HEIGHT = 56;
+    /** The space taken by a smart phone notification bar and digital buttons in dp units. */
+    private static final int SMART_PHONE_HEIGHT = 84; // 56 * 1.5
```

Comment:
Can you expand the comment to a multi-line form  that includes a brief explanation for maintainers on the 1.5 multiplier.  I do not know where that comes from so I'm guessing future maintainers won't either.